src/base/static/state/ducks/places.js

```diff
@@ -22,13 +22,13 @@ export const filteredPlacesSelector = state => {
     return state.places.placeModels;
   }
   // a formId and a location_type are currenty equivalent
-  const filteredFormIds = filters.reduce((memo, filter) => {
-    memo.push(filter.formId);
-    return memo;
-  }, []);
-  return state.places.placeModels.filter(place =>
-    filteredFormIds.includes(place.location_type),
-  );
+  return state.places.placeModels.filter(place => {
+    return !!filters.some(
+      filter =>
+        filter.formId === place.location_type &&
+        filter.datasetSlug === place._datasetSlug,
```

Comment:
@goldpbear Is there a reason why `place._datasetSlug` is prefixed with an underscore? I noticed that we are using `place._datasetSlug` frequently throughout the app. Are there plans to remove it?

Reply:
Is it confusing to have it prefixed? I did that (and also with `_clientSlug`) to indicate that these are properties which shouldn't be sent to the server-- they're client-only properties which get appended to the object which is otherwise sent to the server. 

I guess it's not really needed, though, so if it's confusing we can refactor that out.

Reply:
Yeah, actually I can see the argument that having those specially prefixed properties throughout the code is a distraction. The consuming code really doesn't care if those are client-only properties, and the distinction can be encapsulated in the API client.

I'll refactor that at some point soon.

Reply:
I think this is a strange convention, because in most languages, prefixing an attribute with `_` means that the attribute is private and shouldn't be accessed externally.

As a user of the "place" model, I should not be concerned whether an attribute is sent to the server or not. So yes, I think we should remove the prefix, and include a comment when we strip out the attribute in our api client or serializer.

I'll leave it for now, but add a comment about what the `_` prefix means. But I think we should eventually remove it.